### ISSUE TITLE
WAVE Accessibiltity issues

### ISSUE THREAD

**Issue by atmcgrath** [anvc/scalar#152]
The [WAVE (Web Accessibility Evaluation Tool)](https://wave.webaim.org/) throws up quite a few errors on a basic Scalar site, some of which are the result of application-level issues that cannot be fixed with css. I've developed a few workarounds (see [this checklist](https://github.com/atmcgrath/scalar/blob/master/checklist.md) on my fork) but it would be great if the standard release could address these issues. More robust accessibility testing is probably needed.

**Comment by eloyer** [comment]
Thanks for this feedback and the checklist @atmcgrath , it's very helpful -- will take a look.

**Comment by eloyer** [comment]
@atmcgrath Closing, as recent commits address most issues identified in your checklist — though we will continue to test and make improvements in this area on a regular basis. Note that the large number of ARIA-hidden empty links are due to the way we include RDF data in the page. Since the links are hidden, our understanding is that this won't cause issues for screen readers, and so we're not currently planning to address those items, even though they will continue to be flagged by WAVE.

**Comment by atmcgrath** [comment]
Wonderful, thank you @eloyer!
